ui/src/app/directives/monster-cage-background.directive.ts

```diff
@@ -1,25 +1,25 @@
 import { Directive, ElementRef } from "@angular/core";
 
 
-@Directive ({
-     selector: '[cageBackground]'
+@Directive({
+  selector: '[cageBackground]'
 })
 export class MonsterCageBackgroundDirective {
-  
-  relativePath: string ="../../assets/backgrounds/"
+
+  relativePath: string = "../../assets/backgrounds/"
   href: Array<string> = [
```

Comment:
We should 1) look for public domain images and 2) rename them for easier programmatic loading.